src/main/java/seedu/address/model/tag/ModuleTag.java

```diff
@@ -15,15 +15,47 @@ public class ModuleTag extends Tag implements Comparable<ModuleTag> {
             "NUS Modules should have 2 - 3 letter prefix, followed by 4 digits and optional 1 - 3 alphabets";
     public static final String VALIDATION_REGEX = "[A-Z]{2,4}[0-9]{4}[A-Z]{0,3}";
 
+    private String day;
+    private String startTime;
+    private String endTime;
+    private String location;
+    private String latitude;
+    private String longitude;
+
```

Comment:
Could it also have been `final`?

Reply:
Possible. I am also going to remove location, lat and long.
I will take it as all lessons are in NUS unless we want to include bukit timah campus as well.
Alternative we can have a flag/extra parameter for if the lesson is at BTC

Reply:
Yes alright if it’s too much trouble, then take it as all NUS.

Reply:
Cos I have designed the dataset to have the venue, and it can be mapped to location in venues.txt.